pkg/api/bzz.go

```diff
@@ -0,0 +1,189 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package api
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/ethersphere/bee/pkg/collection/entry"
+	"github.com/ethersphere/bee/pkg/encryption"
+	"github.com/ethersphere/bee/pkg/file"
+	"github.com/ethersphere/bee/pkg/file/joiner"
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/storage"
+	"github.com/ethersphere/bee/pkg/swarm"
+	"github.com/gorilla/mux"
+)
+
+const (
+	ManifestType = "application/bzz-manifest+json"
+)
+
+func (s *server) bzzDownloadHandler(w http.ResponseWriter, r *http.Request) {
+	addressHex := mux.Vars(r)["address"]
+	path := mux.Vars(r)["path"]
+
+	ctx := r.Context()
+
+	address, err := swarm.ParseHexAddress(addressHex)
+	if err != nil {
+		s.Logger.Debugf("bzz download: parse address %s: %v", addressHex, err)
+		s.Logger.Error("bzz download: parse address error")
```

Comment:
No need for error word in error logs.
```suggestion
		s.Logger.Error("bzz download: parse address")
```